README.md

```diff
@@ -1,23 +1,30 @@
 # Pair Programming Lessons Guide/Index
 
-The main reason for this repo to exist is create an index with **topics**, **exercises** and **examples** to be used as guide and reference for people who wants to start learning programming or these whom may want to learn new things or refresh some knowledge for the freecodecamp venezuelan community.
+The main reason for this repository to exist is create a knowledge base, that
+works as guidance to people who wants to introduce themselves to programming and
+web development.
 
-We will also try to describe ideas, tools and software to make this lessons **accessible** and **achievable** for all members, taking into account that some members do not have the best hardware, internet connection or stable electric supply.
+We will also try to share tools that improves the **accessibility** for all
+members who do not have permanent internet connection or good hardware.
 
-It is well known that there are not a real path to learn programming but this guide/index aims to cover simple topics and encourage people to learn with others solving simple exercises doing pair programming or taking a part of their time learning by themselves.
+It is well known that there are not a real path to learn programming but this
+guide/index aims to cover simple topics and encourage people to learn basic
+concepts with others while solving simple exercises in our pair programming
+meetings.
 
-## How to participate in pair programming lessons
+## How to participate in our Pair Programming meetings
 
-We will try to schedule a pair programming exercise every week, free to join to all the participants in our Telegram group:
+We will try to schedule a pair programming exercise every week,
+free to join to all the participants in our Telegram group:
```

Comment:
feel free to join? Or maybe the original intention was something like open and free to join?

I think a word is missing here.